naming/src/main/java/com/alibaba/nacos/naming/core/v2/service/impl/EphemeralClientOperationServiceImpl.java

```diff
@@ -49,6 +49,10 @@ public EphemeralClientOperationServiceImpl(ClientManagerDelegate clientManager)
     public void registerInstance(Service service, Instance instance, String clientId) {
         Service singleton = ServiceManager.getInstance().getSingleton(service);
         Client client = clientManager.getClient(clientId);
+        if (null == client || !client.isEphemeral()) {
+            Loggers.SRV_LOG.warn("Client connection {} already disconnect", clientId);
```

Comment:
The log maybe dismiss maintainer.
It can't include `!client.isEphemeral()`

Reply:
yep.